### PR TITLE
GH:948 tiptap configs

### PR DESCRIPTION
**Pull request by Midnite1212**
Configure tiptap for bible verses and adding notes, including mark down and copy pasting.

### CODE REVIEW

**Review by alvone01 — APPROVED** [review]
Trust Takes Years To Build, Seconds To Break And Forever To Repair - Dhar Mann

**Comment by Midnite1212** [comment]
In my team i trust

File: ui/src/components/helpers/TipTap/BibleVerseExtension.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  parseHTML() {
+    return [
+      {
+        tag: 'bible-verse',
+      },
+    ];
+  },
+
+  renderHTML({ HTMLAttributes }) {
+    return ['bible-verse', mergeAttributes(HTMLAttributes)];
```
Comment (Baeck016) [inline ui/src/components/helpers/TipTap/BibleVerseExtension.js:76]: I don't think you need to put mergeAttributes here cuz you aren't merging any attribute from how you are using it

File: ui/src/components/helpers/TipTap/MenuBar.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    {
+      icon: 'arrow-go-forward-line',
+      title: 'Redo',
+      action: () => editor.chain().focus().redo().run(),
+    },
+    {
+      type: 'divider',
+    },
+    {
+      icon: 'italic',
```
Comment (Baeck016) [inline ui/src/components/helpers/TipTap/MenuBar.js:138]: Maybe can change the icon. Same for adding user sermon note

File: ui/src/components/helpers/TipTap/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import MenuBar from './MenuBar.js';
+import Link from '@tiptap/extension-link';
+import { BibleVerseNode } from './BibleVerseExtension.js';
+import { UserNotesNode } from './UserNotesExtension.js';
+
+export default () => {
+  const editor = useEditor({
+    extensions: [
+      StarterKit.configure({
+        history: false,
```
Comment (Baeck016) [inline ui/src/components/helpers/TipTap/index.js:18]: Undo and Redo button not working because history is false.

**Pull request merged**